backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -31,6 +31,11 @@ body {
     @include margin(null auto);
     max-width: map-get($container-max-widths, "xl");
   }
+
+  &.left-aligned {
```

Comment:
I've updated the `left-aligned` classes (this one, and [the form style](https://github.com/solidusio/solidus/pull/1574/files#diff-b7b0ef96f04e3ccff70950db8525b1deR329)) to be named `max-width`.

This seems to be in line with the function they serve and how they are being discussed in #1290 and other related comments.